.gitignore

```diff
@@ -0,0 +1,24 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+vendor/
```

Comment:
Understandably we don't want vendor files to be part of the git history, but because of the screwed up initial set up of my repo we had to models inside vendor to make it readable. 

This means that combined with this .gitignore models/user.go, is no longer even commited. : /

Reply:
Oops, totally wrong on my part this is fixed.